app/styles/filterbar.scss

```diff
@@ -6,10 +6,76 @@ $heading-adjustment: 2px;
   height: $topbar-height + 2px;
   padding: 0 $spacer-md;
   float: right;
-  .is-map-view &,
+  .is-mobile & {
+    display: none;
+  }
+  .is-desktop.is-map-view &,
   .is-initial-view & {
     width: 50%;
   }
+  &--mobile {
```

Comment:
Sker ret meget her på baggrund af at vi er mobile - tænker det er en god idé at smide et par kommentarer ind der forklarer hvad der skal gøres anderledes på mobil.

Reply:
(det gælder generelt for al stylingen her)